tripal_chado/src/Plugin/TripalStorage/ChadoStorage.php

```diff
@@ -115,36 +115,61 @@ public function addFieldDefinition(string $field_name, object $field_definition)
    * {@inheritDoc}
    * @see \Drupal\tripal\TripalStorage\Interfaces\TripalStorageInterface::getStoredTypes()
    */
-  public function getStoredTypes() {
+  public function getStoredTypes(bool $non_required = FALSE) {
```

Comment:
This change to parameters should not be happening as it does not match the interface... Actually I'm not sure why PHP is not screaming at us for that!

```suggestion
  public function getStoredTypes() {
```